### PR TITLE
Fix the pipeline utils tests

### PR DESCRIPTION
**Pull request by karthikjeeyar**
fixed the pipeline utils test cases

### CODE REVIEW

File: frontend/public/extend/devconsole/shared/utils/__tests__/pipeline-utils.spec.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { mockPipelinesJSON } from '../__mocks__/PipelineUtilMocks';
 
 describe('pipeline-utils ', () => {
   it('For first pipeline there should be 2 stages of length [0:[1],1:[2]]', () => {
-    const stages = getPipelineTasks(mockPipelinesJSON[0].spec.tasks);
+    const stages = getPipelineTasks(mockPipelinesJSON[0]);
     expect(stages.length).toBe(2);
```
Comment (invincibleJai) [inline frontend/public/extend/devconsole/shared/utils/__tests__/pipeline-utils.spec.ts]: instead of `.length` we can make use of `toHaveLength`

**Review by invincibleJai — COMMENTED** [review]
@karthikjeeyar why so many commits, can we squash or fixup

File: frontend/public/extend/devconsole/shared/utils/__tests__/pipeline-utils.spec.ts (hunk 1 above)
Comment (karthikjeeyar) [reply frontend/public/extend/devconsole/shared/utils/__tests__/pipeline-utils.spec.ts]: updated.

**Review by divyanshiGupta — APPROVED** [review]
@karthikjeeyar tests are passing now. Approving the PR.

**Comment by karthikjeeyar** [comment]
@rohitkrai03  @divyanshiGupta @invincibleJai  Sahil has also made the same changes in his PR (https://github.com/talamer/console/pull/137). so closing this PR

**Pull request closed without merging**